scripts/bitcoin.sh

```
@@ -0,0 +1,67 @@
+# -----------------------------------------------------------------------------
+
+bitcoin_cli() {
+  docker_compose exec --user bitcoin bitcoin bitcoin-cli -regtest -rpcport=18443 -rpcuser=bitcoin -rpcpassword=bitcoin "$@"
+}
+
+hdwallet() {
+  docker_compose exec --user bitcoin bitcoin hdwallet "$@"
+}
+
+private_key_from_mnemonic_path() {
+  local mnemonic="${@:1:$#-1}"
```

Shellcheck (0.10.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)